src/content/docs/browser/single-page-app-monitoring/troubleshooting/angular-truncated-copy-paste-snippet.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Angular truncated agent snippet
+type: troubleshooting
+tags:
+  - Browser
+  - Single page app monitoring
+  - Troubleshooting
+---
+
+## Problem
+
+You are using the [HTML copy/paste method](/docs/browser/new-relic-browser/installation-configuration/adding-apps-new-relic-browser/#copy-paste-app) within an Angular version 12, 13, or 14 application, but the agent code snippet has been truncated after running the Angular build.
+
+This may cause the affected application or pages to no longer function correctly due to improper JavaScript contained within the HTML of the page.
+
+## Solution
+
+This issue is caused by a bug within a dependency the Angular CLI build system (versions 12, 13, and 14) rely upon in combination with the browser agent version 1225 and higher. The build has a limitation in the size of a script contained within a script element. We have confirmed this has been corrected with Angular version 15 and recommend upgrading your Angular project.
```

Comment:
(IMO) This first paragraph here kind of flows awkwardly and is a little too specific to scale over time.

```suggestion
This issue is caused by a bug within a dependency the Angular CLI build system. Versions 12, 13, and 14 of the Angular CLI use a dependency that truncates inline scripts of an arbitrary size, which can include the New Relic Browser agent. We have confirmed this has been corrected with Angular version 15 and recommend upgrading your Angular project.
```